README.md

```diff
@@ -31,6 +21,16 @@ This operator was created using the
 [operator-sdk](https://github.com/operator-framework/operator-sdk/). It's not
 strictly required but does provide some handy tooling.
 
+### Knative Serving
+
+The following will install [Knative Serving](https://github.com/knative/serving)
```

Comment:
This applies the CRD and not the CR, and if im not mistaken, it is the CR that allows the operator to install knative